aten/src/ATen/native/native_functions.yaml

```diff
@@ -1890,7 +1890,7 @@
   variants: function, method
   device_guard: False
 
-- func: slogdet(Tensor self) -> (Tensor, Tensor)
+- func: slogdet(Tensor self) -> (Tensor sign, Tensor log)
```

Comment:
Maybe `logabsdet` is a better alternative?

Reply:
@vishwakftw I agree, just renamed.